pkg/ddl/backfilling.go

```diff
@@ -619,6 +621,146 @@ func SetBackfillTaskChanSizeForTest(n int) {
 	backfillTaskChanSize = n
 }
 
+func (dc *ddlCtx) runAddIndexInLocalIngestMode(
+	ctx context.Context,
+	sessPool *sess.Pool,
+	t table.PhysicalTable,
+	reorgInfo *reorgInfo,
+) error {
+	// TODO(tangenta): support adjust worker count dynamically.
+	startKey, endKey := reorgInfo.StartKey, reorgInfo.EndKey
+	if err := dc.isReorgRunnable(reorgInfo.Job.ID, false); err != nil {
+		return errors.Trace(err)
+	}
+	job := reorgInfo.Job.Clone()
```

Comment:
why do we need to clone it?

Reply:
Because in `hasUniqueIndex()`, we call `DecodeArgs`, which destroy `job.Args` with 
```
job.Args = args[:sz]
```

It seems that we can get this information from `indexInfos`. Let me refine the usage by inlining `getBackendCtx`.